tests/unittests/config/test_cc_ansible.py

```diff
@@ -16,6 +16,7 @@
 from tests.unittests.helpers import skipUnlessJsonSchema
 from tests.unittests.util import get_cloud
 
+MOD = "cloudinit.config.cc_ansible."
```

Comment:
Nit: According to our [development documentation](https://cloudinit.readthedocs.io/en/latest/topics/testing.html), it would be nice to name this as `M_PATH`.